docs/hardhat-network/reference/README.md

```diff
@@ -109,6 +109,28 @@ networks: {
 
 This means that no new blocks will be mined by the Hardhat Network, but you can manually mine new blocks using the `evm_mine` RPC method. This will generate a new block that will include as many pending transactions as possible.
 
+#### Transaction ordering
+
+Hardhat Network can sort mempool transactions in two different ways. How they are sorted will alter which transactions from the mempool get included in the next block, and in which order.
```

Comment:
I don't like that here i'm using "sorting" all the time, and then the setting is "order". 

Any rewording addressing this will be welcome :)

Reply:
I think this paragraph in particular is just fine, but I hear what you're saying and I'm offering a few tweaks to the subsequent paragraphs.